src/mission_execution/include/mission_execution/MissionExecutor.hpp

```diff
@@ -30,7 +30,7 @@ class MissionExecutor : public nav2_tasks::ExecuteMissionTaskServer
   MissionExecutor() = delete;
   ~MissionExecutor();
 
-  nav2_tasks::TaskStatus executeAsync(
+  nav2_tasks::TaskStatus execute(
```

Comment:
I changed the name since it's not asynchronous on the server side (it is asynchronous on the client side and I kept 'executeAsync' there).